community/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/BuiltInProcedureAcceptanceTest.scala

```diff
@@ -204,6 +204,6 @@ class BuiltInProcedureAcceptanceTest extends ProcedureCallAcceptanceTest {
 
     // Then
     result.toList should equal(
-      List(Map("description" -> "INDEX ON :A(prop)", "state" -> "ONLINE")))
+      List(Map("description" -> "INDEX ON :A(prop)", "state" -> "ONLINE", "type" -> "node_label_property")))
```

Comment:
Why don't you use the enum constants in this assertion? The test will fail if the enum is renamed.